.goreleaser.yml

```
@@ -6,7 +6,7 @@ monorepo:
 builds:
   - main: ./cmd/flipt/.
     ldflags:
-      - -s -w -X main.version={{ .Version }} -X main.commit={{ .Commit }} -X main.date={{ .Date }} -X main.analyticsKey={{ .Env.ANALYTICS_KEY }}
+      - -s -w -X main.version=v{{ .Version }} -X main.commit={{ .Commit }} -X main.date={{ .Date }} -X main.analyticsKey={{ .Env.ANALYTICS_KEY }}
```

Alternatively, `.PrefixedTag` ([ref](https://goreleaser.com/customization/templates/)) _might_ work as well, but I haven't used that field before, so not sure.

E.g.
```suggestion
      - -s -w -X main.version={{ .PrefixedTag }} -X main.commit={{ .Commit }} -X main.date={{ .Date }} -X main.analyticsKey={{ .Env.ANALYTICS_KEY }}
```

---

I think you might be right with that suggestion there.
Lets give `.PrefixedTag` ago @dstotijn 👍